File: app/src/main/java/com/developer/valyutaapp/ui/widget/WidgetViewInterface.java
package com.developer.valyutaapp.ui.widget;


import com.developer.valyutaapp.model.Valute;

import java.util.List;

public interface WidgetViewInterface {
    void showToast(String s);
    void showProgressBar();
    void hideProgressBar();
    void displayValuteWithId(Valute valute);
    void displayValutes(List<Valute> valute);
    void displayError(String s);
}

File: app/src/main/java/com/developer/valyutaapp/ui/widget/WidgetViewInterface.kt
package com.developer.valyutaapp.ui.widget

import com.developer.valyutaapp.domain.entities.Valute

interface WidgetViewInterface {
    fun showToast(s: String)
    fun showProgressBar()
    fun hideProgressBar()
    fun displayValuteWithId(valute: Valute)
    fun displayValutes(valute: List<Valute>)
    fun displayError(s: String)
}